src/mods.py

```diff
@@ -24,6 +24,7 @@
     '--uac-admin',
     '--uac-uiaccess',
     '--noupx',
+    '--noconfirm',                      '-y',
```

Comment:
I want to use it by default, so its not necessary to be passed as input

Reply:
i'll ajust it right away

Reply:
Done